pyproject.toml

```diff
@@ -2,17 +2,10 @@
 requires = [
     "setuptools >= 41.0.0",
     "wheel >= 0.30.0",
-    "setuptools_scm[toml]>=3.4",
+    "setuptools_scm >= 2",
 ]
 build-backend = 'setuptools.build_meta'
 
-[tool.setuptools_scm]
```

Comment:
Sadly using the toml config degrades really bad on old platforms, so we've dropped it's usage in line with https://github.com/pypa/setuptools_scm#setuppy-usage

Reply:
@pradyunsg 😢